backend/app.py

```diff
@@ -204,21 +204,21 @@ def api_get_disk_by_disk_id(disk_id):
 def api_get_disk_by_poll_id(poll_id):
     return disks.get_disk_by_poll_id(poll_id)
 
-@app.route('/api/network', methods=['GET'])
+@app.route('/api/networks', methods=['GET'])
```

Comment:
this is the only line that affects me, taking note

Reply:
thanks!